### DISCUSSION TITLE
各位的3.2版本有全选的功能吗？

### DISCUSSION THREAD

**Discussion by zhujitop** [AlistGo/alist#2019] [Q&A]
前后台看了一圈。
怎么都没有看见 全选的 选项？

全选 一键复制所有文件的直链。

还是2.0的方便，不用先打开功能，直接右上角就有个“复制直链”，一下默认就复制全部了。

3.0还要先点 更多 - 打开复选框 - 然后全选 -再来复制。。
关键是我咋还没看见全选...
怎么搞出来的？

https://pic.iqy.ink/i/2022/10/14/634972975ddb1.jpg
https://pic.iqy.ink/i/2022/10/14/634974250c2e1.jpg

终于知道了。。
必须还得先改成列表方式才有个点“名称”的选项，才能一次性默认复制全部直链。

这步骤。。。
比2.0多走好几步。

2.0： 直接在文件夹里面点1次右上角的“复制直链”
3.0：点“名称” - 点下面工具栏“复制链接” - 再点弹出来的“下载链接” （不需要预览并且默认是列表展示和开启复选框）
最快都要点3次，如果需要预览，还得再切换成网格布局。

**Comment by ma3252788** [comment]
![image](https://user-images.githubusercontent.com/22229456/202397442-631d63ee-1dc8-4451-ba0a-75a492af7f20.png)
这个不就是么，名称左边
